Wikipedia/Code/WMFExploreSectionSchema.m

```diff
@@ -400,7 +400,7 @@ - (nullable WMFExploreSection*)existingNearbySection {
 
     WMFExploreSection* latestMostReadSection = [self newMostReadSectionWithLatestPopulatedDate];
 
-    BOOL containsLatestSectionEquivalent = [mostReadSections bk_match:^BOOL (WMFExploreSection* mostReadSection) {
+    BOOL containsLatestSectionEquivalent = [mostReadSections bk_any:^BOOL (WMFExploreSection* mostReadSection) {
```

Comment:
yeah, this is the warning I'd like to trigger in debug mode :confounded: don't know why it's only showing up in release. oh well, at least we caught it